src/Sarif.Converters/MSBuildConverter.cs

```diff
@@ -83,9 +84,9 @@ private IList<Result> GetResults(TextReader reader)
             \s*:\s*
             (?<qualifiedLevel>
               (?<levelQualification>.*)        # For example, 'fatal'.
-              (?<level>error|warning|note|info|pass|review|open|notapplicable)
+              (?<level>error|err|warning|wrn|note|info|pass|review|open|notapplicable)
```

Comment:
>wrn [](http://example.com/codeflow?start=41&length=3)

just to get info, the reason we need `wrn` and `err` ?

Reply:
Don't see a definition of what the level should be in .ERR file. Have seen "Error"/"Warning" also the abbreviation forms "ERR"/"WRN" in build logs. Here just try to match the keyword as much as possible